### ISSUE TITLE
The plugin doesn't work 

### ISSUE THREAD

**Issue by eran-or** [MaxKless/git-squash-all#1]
I installed the plugin.
run git rebase -i HEAD~3 for example.
command + p ,shift + >, git squash all
nothing happened . 
I tried to open the vs code again , abort and rebase again. nothing happens.
In the past on another computer it worked before 4 months. 

I'm running mac pro 2013 if it helps.

**Comment by MaxKless** [comment]
Hey thanks for reporting this. When you run the rebase, does it open another window?

**Comment by eran-or** [comment]
no, nothing happened. I can't say more, I deleted the plugin for now until
it will be fixed. Maybe you should do a youtube video on how to use it, or
maybe I missed something, don't know.

‫בתאריך יום ד׳, 2 בנוב׳ 2022 ב-20:28 מאת ‪MaxKless‬‏ <‪
***@***.***‬‏>:‬
